Orbit.h

```diff
@@ -92,10 +90,10 @@ ORBIT_STUB void TrackDoubleAsInt64(const char*, int64_t) { ORBIT_NOOP(); }
 // Convert floating point arguments to integer arguments as we can't access
 // XMM registers with our current dynamic instrumentation on Linux (uprobes).
 inline void TrackFloat(const char* name, float value) {
-  TrackFloatAsInt(name, absl::bit_cast<int32_t>(value));
+  TrackFloatAsInt(name, *reinterpret_cast<int32_t*>(&value));
```

Comment:
In which case it is better to use memcpy here.

int int_value;
static_assert(sizeof(value == sizeof(int_value);
memcpy(&int_value, &value, sizeof(value));
TrackFloatAsInt(...);

Reply:
Yeah, I would also vote for std::memcpy. All major compilers optimize that away, so there are no downsides and it has the advantage that it's not UB.

Apart from that. Thanks for fixing this!

Reply:
Cool, done, thanks!